junifer/data/tests/test_template_spaces.py

```diff
@@ -0,0 +1,30 @@
+"""Provide tests for template spaces."""
+
+# Authors: Synchon Mandal <s.mandal@fz-juelich.de>
+# License: AGPL
+
+import socket
+from pathlib import Path
+
+import pytest
+
+from junifer.data import get_xfm
+
+
+@pytest.mark.skipif(
+    socket.gethostname() != "juseless",
```

Comment:
Why only for juseless?

Reply:
The download takes well around 10 mins.

Reply:
damn, we need to create a "slow test" suite that we use it to check only when about to merge or something like that. For the moment we keep it like this.

Reply:
Yeah that's a good idea. Well the essential issue is that Gin is not really as powerful as GitHub when you consider infrastructure, so the octet-stream downloads are not cached from what I understand.